File: Computer-Seekho-Java/ComputerSeekho/src/main/java/com/Project/Entities/Student.java
package com.Project.Entities;
import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDate;

@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
@Table(name = "student")
public class Student {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "student_id")
    private int studentId;

    @Column(name = "payment_due")
    private int paymentDue;

    @Column(name = "student_name", length = 30, nullable = false)
    private String studentName;

    @Column(name = "student_address", length = 60)
    private String studentAddress;

    @Column(name = "student_gender", length = 10)
    private String studentGender;

    @Column(name = "photo_url", length = 255)
    private String photoUrl;

    @Column(name = "student_dob")
    private LocalDate studentDob;

    @Column(name = "student_qualification", length = 20)
    private String studentQualification;

    @Column(name = "student_mobile")
    private String studentMobile;

    @Column(name = "student_email", length = 30)
    private String studentEmail;

    @ManyToOne
    @JoinColumn(name = "batch_id", referencedColumnName = "batch_id")
    private Batch batch;

    @ManyToOne
    @JoinColumn(name = "course_id", referencedColumnName = "course_id")
    private Course course;
}

File: Computer-Seekho-.NET/Computer_Seekho_DN/Models/Student.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text.Json.Serialization;
using Microsoft.EntityFrameworkCore;

namespace Computer_Seekho_DN.Models;

[Table("student")]
[Index("BatchId", Name = "FK17mfv6a26cwnmli2b6vm00dn7")]
[Index("CourseId", Name = "FKdfypyqt0stgfc0aij9kcxm99s")]
public partial class Student
{
    [Key]
    [Column("student_id")]
    public int StudentId { get; set; }

    [Column("payment_due")]
    public int PaymentDue { get; set; }

    [Column("photo_url")]
    [StringLength(255)]
    public string? PhotoUrl { get; set; }

    [Column("student_address")]
    [StringLength(60)]
    public string? StudentAddress { get; set; }

    [Column("student_dob")]
    public DateOnly? StudentDob { get; set; }

    [Column("student_email")]
    [StringLength(30)]
    public string? StudentEmail { get; set; }

    [Column("student_gender")]
    [StringLength(10)]
    public string? StudentGender { get; set; }

    [Column("student_mobile")]
    [StringLength(255)]
    public string? StudentMobile { get; set; }

    [Column("student_name")]
    [StringLength(30)]
    public string StudentName { get; set; } = null!;

    [Column("student_qualification")]
    [StringLength(20)]
    public string? StudentQualification { get; set; }

    [Column("batch_id")]
    public int? BatchId { get; set; }

    [Column("course_id")]
    public int CourseId { get; set; }

    [ForeignKey("BatchId")]
    [InverseProperty("Students")]
    public virtual Batch? Batch { get; set; }

    [ForeignKey("CourseId")]
    [InverseProperty("Students")]
    public virtual Course? Course { get; set; }

    [InverseProperty("Student")]
    [JsonIgnore]
    public virtual ICollection<Payment> Payments { get; set; } = new List<Payment>();

    [InverseProperty("Student")]
    [JsonIgnore]
    public virtual Placement? Placement { get; set; }
}
